Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/ActiveEvent.cs

```diff
@@ -61,6 +61,12 @@ public GameObject AudioEmitter
             private set;
         }
 
+        public string MessageToSend
```

Comment:
I did it using unity messages to follow the unity model for this sort of thing, which I've noticed the rest of the HoloToolkit does.  Examples of the top of my head are "OnGazeEntered", "OnSelect", but know there are a bunch more.  Are there any examples in the HoloToolkit where someone follows the .net event pattern instead of the unity message pattern?